src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -16,8 +16,10 @@ public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
     }
 
 
-    public List<Video> findAllVideos() throws VideoNotFoundException {
```

Comment:
On line 11 change the CultureMediaServiceImpl to CultureMediaService since the CultureMediaController class has no direct contact with CultureMediaServiceImpl